superdesk/services.py

```diff
@@ -122,6 +122,9 @@ def get_from_mongo(self, req, lookup, projection=None):
             req.projection = json.dumps(projection)
         return self.backend.get_from_mongo(self.datasource, req=req, lookup=lookup)
 
+    def get_all(self):
```

Comment:
If the collection is too big, would this result in an OOM being raised? Would it be better to use generators that yield pages for consumers to iterate over all documents?

Reply:
it's using batches internally in mongo cursor, I tested it on the bi media db and it works fine